File: OopsFundamentals/classesAndObject/java/ClassesAndObjects.java
package OopsFundamentals.classesAndObject.java;
class Car
{
    private String Brand;
    private String model;
    private double speed;

    public Car(String brand,String model)
    {
        this.Brand = brand;
        this.model = model;
        this.speed = 0.0;

    }
    public void accelerate(double accSpeed)
    {
        speed += accSpeed;
    }
    public void display()
    {
        System.out.println("The car of brand "+Brand+" and model "+ model+ " is running at speed : "+speed+" KM/H");
    }
}

class ClassesAndObjects
{
    public static void main(String[] args) 
    {
        Car corolla = new Car("Toyota", "CORROLA");
        Car mustang = new Car("Ford", "Mustang");

        corolla.accelerate(250);
        mustang.accelerate(300);

        corolla.display();
        mustang.display();

    }
}

File: OopsFundamentals/classesAndObject/python/classes_and_objects.py
class Car:
    def __init__(self, brand, model):
        self.brand = brand
        self.model = model
        self.speed = 0.0

    def accelerate(self, acc_speed):
        self.speed += acc_speed

    def display(self):
        print(f"The car of brand {self.brand} and model {self.model} is running at speed : {self.speed} KM/H")


class ClassesAndObjects:
    @staticmethod
    def main():
        corolla = Car("Toyota", "COROLLA")
        mustang = Car("Ford", "Mustang")

        corolla.accelerate(250)
        mustang.accelerate(300)

        corolla.display()
        mustang.display()


if __name__ == "__main__":
    ClassesAndObjects.main()
